models/ed/R/write.configs.ed.R

```diff
@@ -382,12 +382,15 @@ write.config.xml.ED2 <- function(settings, trait.values, defaults=settings$const
 
       ## Convert trait values to ED units
       converted.trait.values <- convert.samples.ED(trait.values[[i]])
-
+      
       ## Selectively replace defaults with trait values
       vals <- modifyList(vals, converted.trait.values)
 
+      ## Convert settings constants to ED units
+      converted.defaults <- convert.samples.ED(defaults[[pft]])
```

Comment:
I inserted this line before the conversion

```
names(defaults) <- sapply(defaults, function(x) x$name)
```

and then instead of defaults[[pft]] I wrote defaults[[pft]]$constants 
```
converted.defaults <- convert.samples.ED(defaults[[pft]]$constants)
```
and everything worked for me (I mean with the workflow). Did we have a specific reason to get rid of that [line](https://github.com/PecanProject/pecan/blob/master/models/ed/R/write.configs.ed.R#L158)?